src/Symfony/Bundle/FrameworkBundle/DependencyInjection/FrameworkExtension.php

```diff
@@ -1445,6 +1445,22 @@ private function registerMessengerConfiguration(array $config, ContainerBuilder
 
         $container->getDefinition('messenger.sender_locator')->replaceArgument(0, $senderLocatorMapping);
         $container->getDefinition('messenger.asynchronous.routing.sender_locator')->replaceArgument(1, $messageToSenderIdsMapping);
+
+        foreach ($config['adapters'] as $name => $adapter) {
+            $container->setDefinition($adapterId = 'messenger.'.$name.'_adapter', (new Definition(\Symfony\Component\Messenger\Adapter\Factory\AdapterInterface::class, array(
+                $adapter['dsn'],
+            )))->setFactory(array(new Reference('message.adapter_factory'), 'create'))->setPublic(true));
```

Comment:
Shouldn't it use the short syntax for arrays here?

Reply:
No, symfony doesn't use short array syntax yet for maintenance reasons.